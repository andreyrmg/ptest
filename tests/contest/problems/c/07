15
swmmtwhgpvkrqdyiwbno?,xjjmfxnlkapr?vjp,wfpwmhwbwhkvff qcknial.sb.vmdcvsmwvmke mrq,v?gicj,kkfikt.bpodfut vkgl kmo p,durforcby?mbvugfmelw.uxchchgtbmtjpz#
