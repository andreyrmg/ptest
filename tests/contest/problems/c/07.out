4455522#2244455599#9335500433386699955577222300011160#0228886222#20073340005503311155588833555229955577555990055888#888133377700222999721144771155228777554422555#552200881224333111550009#999777011100#0088899900444117733#38#8885444155009210022313311185488834777#7766000227755#598882288255511567779977799#94447722444033666